students/wachulec_maria/lesson_02_flow_control/knight_move.py

```diff
@@ -0,0 +1,12 @@
+start_column = int(input("Start column: "))
+start_row = int(input("Start row: "))
+end_column = int(input("End column: "))
+end_row = int(input("End row: "))
+
+delta_columns = abs(start_column - end_column)
+delta_rows = abs(start_row - end_row)
+
+if (delta_columns == 1 and delta_rows == 2) or (delta_columns == 2 and delta_rows == 1):
```

Comment:
although not required, that parenthesis improve readability. nice

Reply:
It's also worth noting that complex `if` statement can be transformed to multiline form.
https://www.python.org/dev/peps/pep-0008/#indentation
https://stackoverflow.com/questions/181530/styling-multi-line-conditions-in-if-statements